resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/config.scala

```diff
@@ -70,4 +70,19 @@ package object config {
           "during a temporary disconnection, before tearing down all the executors.")
       .doubleConf
       .createWithDefault(0.0)
+
+  private [spark] val NETWORK_NAME =
```

Comment:
Tiny nit, could you remove the space after `private`? that would make it consistent with the rest of the code.

I trust your judgment as to whether this is the right functional change for Mesos and the details and style look OK to me otherwise.